### ISSUE TITLE
After running your code, I got poor results. I want to ask what I did wrong?

### ISSUE THREAD

**Issue by xinhaoH** [IC3Net/IC3Net#9]
I set the same seed for numpy and pytorch.
    torch.manual_seed(args.seed)
    np.random.seed(args.seed)

The results I got are as follows:
![ic3net_7715](https://user-images.githubusercontent.com/39089809/111410087-0d2b4500-8713-11eb-8258-6b2ac4b11a82.png)
![ic3net_0 27_1314_vision3](https://user-images.githubusercontent.com/39089809/111410111-1ae0ca80-8713-11eb-9c70-73f2fbcc16cc.png)

python main.py --env_name traffic_junction --nagents 10 --nprocesses 1 --num_epochs 2000 --hid_size 128 --detach_gap 10 --lrate 0.003 --dim 14 --max_steps 40 --ic3net --recurrent --vision 0 --add_rate_min 0.05 --add_rate_max 0.2 --curr_start 250 --curr_end 1250 --difficulty medium --seed 7715

python main.py --env_name traffic_junction --nagents 10 --nprocesses 1 --num_epochs 2000 --hid_size 128 --detach_gap 10 --lrate 0.003 --dim 14 --max_steps 40 --ic3net --recurrent --**vision 3** --add_rate_min 0.05 --add_rate_max 0.2 --curr_start 250 --curr_end 1250 --difficulty medium --seed 1314
